Formalisation(shacl)/Core/PiecesShape/DatasetSeries.ttl

```diff
@@ -86,7 +86,9 @@
   [
     sh:path dct:type ;
     sh:nodeKind sh:IRI ;
-    sh:minCount 1 ;
+    #sh:minCount 1 ;
```

Comment:
Do we want to keep this commented out, or to remove the line? I'd be inclined to use comments only for explanations or descriptions of choices if needed. We have the git history for keeping track of what happened.

Reply:
done and will check other shapes in another ticket.